### PR TITLE
Remove LTSS repositories before migration (bsc#1173306)

### PR DESCRIPTION
**Pull request by tbazant**
LTSS repos break migration form SES5

### CODE REVIEW

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -485,6 +485,24 @@ mds standby for name = mds.456-another-mds
     Check that required repositories are configured on each cluster's node. To
```
Comment (lproven) [inline xml/admin_ceph_upgrade.xml:485]: > each cluster's node

...(?)

What does this use of node in the singular mean? 1 node in each? All nodes in each cluster seems more likely to me, but I am not sure. If so, how about "each cluster's nodes"?

File: xml/admin_ceph_upgrade.xml [hunk 2]
```diff
@@ -485,6 +485,24 @@ mds standby for name = mds.456-another-mds
...
     list all available repositories, run
    </para>
+<screen>
+&prompt.sminion;zypper lr
+</screen>
+<important>
+ <title>Remove &productname; &prevproductnumber; LTSS Repositories</title>
+ <para>
+  Upgrade will fail if LTSS repositories are present in the &productname; &prevproductnumber;.
+  Find their ID and remove them from the system, for example:
```
Comment (lproven) [inline xml/admin_ceph_upgrade.xml:495]:
```suggestion
  Find their IDs and remove them from the system; for example:
```

File: xml/admin_ceph_upgrade.xml [hunk 3]
```diff
@@ -485,6 +485,24 @@ mds standby for name = mds.456-another-mds
     Check that required repositories are configured on each cluster's node. To
     list all available repositories, run
    </para>
+<screen>
+&prompt.sminion;zypper lr
+</screen>
+<important>
+ <title>Remove &productname; &prevproductnumber; LTSS Repositories</title>
+ <para>
+  Upgrade will fail if LTSS repositories are present in the &productname; &prevproductnumber;.
```
Comment (lproven) [inline xml/admin_ceph_upgrade.xml:494]:
```suggestion
  Upgrades will fail if LTSS repositories are configured in &productname; &prevproductnumber;.
```

File: xml/admin_ceph_upgrade.xml (hunk 1 above)
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:485]:
```suggestion
    Check that required repositories are configured on each host of the cluster. To
```

Comment (tbazant) [reply xml/admin_ceph_upgrade.xml]: fixed

**Pull request merged**